### PR TITLE
Use message timestamp serializer from Pipeline-Infrastructure

### PR DESCRIPTION
**Pull request by as2388**
Requires https://github.com/AfricasVoices/Pipeline-Infrastructure/pull/63.

Fixes some consistency and formatting issues along the way.

### CODE REVIEW

**Review by IsaackMwenda — APPROVED** [review]

File: src/engagement_db_to_analysis/cache.py [hunk 1]
```diff
@@ -48,38 +46,41 @@ def set_latest_message_timestamp(self, engagement_db_dataset, last_updated):
...
+    def set_messages(self, engagement_db_dataset, messages):
         """
-        Exports a list of messages for the given engagement_db_dataset to cache.
+        Sets a list of messages for the given engagement_db_dataset.
 
         :param engagement_db_dataset: Engagement db dataset name for this context.
         :type engagement_db_dataset: str
-        :return: list of messages
-        :rtype: list
+        :param messages: Messages to set.
```
Comment (DanielMwendwa) [inline src/engagement_db_to_analysis/cache.py:80]: "Messages to set for the given engagement_db_dataset". I think this avoids thinking of type set

**Review by DanielMwendwa — APPROVED** [review]

File: src/engagement_db_to_analysis/engagement_db_to_analysis.py [hunk 2]
```diff
@@ -90,34 +77,36 @@ def _get_project_messages_from_engagement_db(analysis_configurations, engagement
...
     return engagement_db_dataset_messages_map
 
+
 def _convert_messages_to_traced_data(user, messages_map):
     """
     Converts messages dict objects to TracedData objects.
 
     :param user: Identifier of user running the pipeline.
     :type user: str
     :param messages_map: Dict containing messages data.
```
Comment (DanielMwendwa) [inline src/engagement_db_to_analysis/engagement_db_to_analysis.py:91]: engagement_db_dataset_messages_map of engagement_db_dataset to list of messages.

File: src/engagement_db_to_analysis/cache.py (hunk 1 above)
Comment (as2388) [reply src/engagement_db_to_analysis/cache.py:80]: updated

File: src/engagement_db_to_analysis/engagement_db_to_analysis.py (hunk 2 above)
Comment (as2388) [reply src/engagement_db_to_analysis/engagement_db_to_analysis.py:91]: Thanks, I see now that this doc was very unclear so have fixed

**Pull request merged**